tests/test_atom_mapping.py

```diff
@@ -542,6 +546,7 @@ def test_linker_map():
         disallow_planar_torsion_flips=False,
         min_threshold=0,
         initial_mapping=None,
+        enforce_chirally_valid_dummy_groups=False,  # TODO: understand why assertion fails when set to True
```

Comment:
I think we could either set this to `False` or update the assertion (the assertion seems to assume a core that has no chirally-valid dummy group assignments). @proteneer does this seem right?

Reply:
This is a test case devoid of any hydrogens, and `chiral_utils.find_chiral_atoms` fails for molecules that have implicit hydrogens. There's no sensical way to interpret chirality for implicit hydrogens.

Reply:
Added a note (and also set `enforce_chiral=False`) in this example in https://github.com/proteneer/timemachine/pull/1356/commits/8ef161c71506397358a7d3a34bc60c69f04d7c2c